packages/core/src/components/AccordionGroup/Accordion.tsx

```diff
@@ -0,0 +1,31 @@
+import CardBase from '@components/Card/CardBase';
+import { AccordionProps } from './types';
+import { createWrapperStyles } from './styles';
+
+const Accordion = ({
+  title,
+  isActive,
+  ariaControls,
+  tabId,
+  variant = 'empty',
+  renderTitle,
+  renderContent,
+  onClick,
+}: AccordionProps) => {
+  const styles = createWrapperStyles();
+  return (
+    <CardBase
+      role="tab"
```

Comment:
Is this the correct role for our case? I see in this example `role="region"` - https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

Reply:
Done